server/server.go

```diff
@@ -116,7 +116,7 @@ type Server struct {
 
 	supportedResponseTypes map[string]bool
 
-	discoveryAllowedOrigins []string
+	allowedOrigins []string
```

Comment:
since this is only used when setting up the handlers I don't think we need this on the server.